spec/models/recipe_spec.rb

```diff
@@ -19,4 +19,14 @@
       expect(recipe).not_to be_valid
     end
   end
+
+  context 'methods' do
+    it 'should return the number of food items' do
+      expect(recipe.count_food_recipes).to eq(0)
+    end
+
+    it 'should return the sum of food items' do
+      expect(recipe.sum_foods).to eq(0)
+    end
```

Comment:
Would it make sense to test a case where there are food items? This would give the reader a better sense of what these methods do.